assets/base.css

```diff
@@ -3120,3 +3120,41 @@ details-disclosure > details {
 .rte blockquote > * {
   margin: -0.5rem 0 -0.5rem 0;
 }
+
+/* Animations */
+
+.scroll-animation__trigger {
+  transition: opacity var(--duration-long) ease-out;
+  opacity: 0;
+}
+
+.scroll-animation__trigger--active {
+  animation: var(--duration-long) ease-out slideIn forwards;
+}
+
+@keyframes slideIn {
+  from {
+    transform: translateY(10vh);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+/* Turn off animations if JS is turned off. */
+.no-js .scroll-animation__trigger,
+.no-js .scroll-animation__trigger--active {
+  opacity: 1;
+  animation: none;
+}
+
+/* Respect the visitor's motion preferences. */
+@media (prefers-reduced-motion) {
```

Comment:
I think this was based on our original plans to enable animations by default, but we don't have a final decision on this yet. Since we default to animations, unless the user has actively done something to turn them off, I find it more readable to follow the pattern where we use `@media (prefers-reduced-motion)`, but I don't have a strong opinion on it either way and I'm happy to change it